### PR TITLE
Replace fs.exists with other fileSystem alternatives

### PR DESCRIPTION
**Pull request by Shriram-Balaji**
**Type:** chore
**Scope:** plugins/local-storage

The following has been addressed in the PR:

*  There is a related issue? https://github.com/verdaccio/monorepo/issues/60
*  Unit or Functional tests are included in the PR - Yes

**Description:**

This PR removes our usage of `fs.exists` and replaces it with other file system API  alternatives like `fs.access` and `fs.stat`.

Resolves https://github.com/verdaccio/monorepo/issues/60

### CODE REVIEW

**Comment by juanpicado** [comment]
@sergiohgz I think the token just expired, it failed on publish

File: plugins/local-storage/tests/local-fs.test.ts [hunk 1]
```diff
@@ -182,8 +182,8 @@ describe('Local FS test', () => {
 
       writeTarballStream.on('success', function() {
         const fileLocation: string = path.join(__dirname, '../_storage', newFileName);
+        expect(() => fs.accessSync(fileLocation, fs.constants.R_OK | fs.constants.W_OK)).not.toThrow();
```
Comment (DanielRuf) [inline plugins/local-storage/tests/local-fs.test.ts]: Please only change `exists` to an alternative or `existsSync`.

Comment (Shriram-Balaji) [reply plugins/local-storage/tests/local-fs.test.ts]: @DanielRuf Okay, I can make that change. But technically fs.exists also uses fs.access internally.

Comment (DanielRuf) [reply plugins/local-storage/tests/local-fs.test.ts]: This piece of code already uses `existsSync` so refactoring it is not necessary.

File: plugins/memory/src/memory-handler.ts [hunk 2]
```diff
@@ -106,8 +106,8 @@ class MemoryHandler implements IPackageStorageManager {
     const temporalName = `/${name}`;
 
     process.nextTick(function() {
-      fs.exists(temporalName, function(exists) {
-        if (exists) {
+      fs.stat(temporalName, function(fileError, stats) {
+        if (!fileError && stats) {
```
Comment (DanielRuf) [inline plugins/memory/src/memory-handler.ts]: Why `!fileError`? When it is set there is an error. If it is not set set (== false) there is no error.

File: plugins/local-storage/tests/local-fs.test.ts (hunk 1 above)
Comment (Shriram-Balaji) [reply plugins/local-storage/tests/local-fs.test.ts]: @DanielRuf That makes sense, I've made the change and update the PR.

**Comment by juanpicado** [comment]
Thanks guys !!

**Pull request merged**